Rakefile

```diff
@@ -87,3 +87,54 @@ namespace :students do
     end
   end
 end
+
+namespace :comments do
+  task :migrate do
+    Mongo::Logger.logger.level = ::Logger::FATAL
+    module Classroom::Collection::Comments
+      extend Mumukit::Service::Collection
+      def self.mongo_collection_name
+        :comments
+      end
+      def self.mongo_database
+        Classroom::Database
+      end
+    end
+    class Classroom::Collection::ExerciseStudentProgress < Classroom::Collection::CourseCollection
```

Comment:
Not yet! I'm still migrating the data

El mar., 13 de sep. de 2016 a la(s) 15:12, Franco Leonardo Bulgarelli <
notifications@github.com> escribió:

> In Rakefile
> <https://github.com/mumuki/mumuki-classroom-api/pull/57#discussion_r78614102>
> :
>
> > @@ -87,3 +87,54 @@ namespace :students do
> >      end
> >    end
> >  end
> > +
> > +namespace :comments do
> > +  task :migrate do
> > +    Mongo::Logger.logger.level = ::Logger::FATAL
> > +    module Classroom::Collection::Comments
> > +      extend Mumukit::Service::Collection
> > +      def self.mongo_collection_name
> > +        :comments
> > +      end
> > +      def self.mongo_database
> > +        Classroom::Database
> > +      end
> > +    end
> > +    class Classroom::Collection::ExerciseStudentProgress < Classroom::Collection::CourseCollection
>
> @aguspina <https://github.com/aguspina> @fedescarpa
> <https://github.com/fedescarpa> have you already migrated those comments?
> Can we remove this code now?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/mumuki/mumuki-classroom-api/pull/57/files/32e4d4158fe37dfa294ac561aedaeefb839ebe0a#r78614102>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD6mDHaVls2hQL2cgyH7BPAzu6zf289Eks5qpud5gaJpZM4J75I7>
> .
>